rj_gameplay/rj_gameplay/tactic/pass_seek.py

```diff
@@ -0,0 +1,94 @@
+"""Contains the stub for the move tactic. """
+
+from dataclasses import dataclass
+from typing import List, Optional
+from typing import Dict, Generic, List, Optional, Tuple, Type, TypeVar
+
+import stp.action as action
+import stp.rc as rc
+import stp.tactic as tactic
+import stp.role as role
+import stp.utils.pass_seeker_optimizer as optimizer
+
+import rj_gameplay.eval
+import rj_gameplay.skill as skills
+from rj_gameplay.skill import shoot, capture, move
+import stp.skill as skill
+import numpy as np
+
+def seek_heuristic(point: Tuple[float, float], world_state: Tuple[rc.WorldState]) -> float:
+    cost = 0
+    for robot in world_state.their_robots:
+        cost-= np.linalg.norm(np.array(point)-np.array(robot.pose[0:2]))
+    cost -= 7 * point[1]
+    return cost
+
+class seek_cost(role.CostFn):
+    """
+    A cost function for how to choose a seeking robot
+    TODO: Implement a better cost function
+    """
+    def __init__(self, target_point : np.ndarray):
+        self.target_point = target_point
+
+    def __call__(
+        self,
+        robot: rc.Robot,
+        prev_result: Optional["RoleResult"],
+        world_state: rc.WorldState,
+    ) -> float:
+    
+        if robot.id == 7:
```

Comment:
I assume you're filling in this cost func soon, but make this a temp tactic if you aren't.

Reply:
its a test tactic, I've made all the tactics related to passing take in cost functions in their inits

Reply:
awesome